hosted/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM 471112759938.dkr.ecr.us-west-2.amazonaws.com/docker.io/sci/library/ubuntu:22.04
```

Comment:
Utilizes our ECR pull through cache so we don't accidentally hit dockerhub rate limit issus. Also allows us to do security scanning and other security monitoring on the images we pull.